pyro/contrib/timeseries/gp.py

```diff
@@ -422,3 +426,58 @@ def forecast(self, targets, dts):
         filtering_state = self._filter(targets)
         predicted_mean, predicted_covar = self._forecast(dts, filtering_state)
         return MultivariateNormal(predicted_mean, predicted_covar)
+
+
+class LinearlyCoupledDependentMaternGP(DependentMaternGP):
+    """
+    A time series model in which each output dimension is modeled as a linear combination
+    of univariate Gaussian Processes with Matern kernels. The different output dimensions
+    are additionally correlated because the Gaussian Processes are driven by a correlated
+    Wiener process. That is, this time series model combines the structure of the
+    `LinearlyCoupledMaternGP` (with `num_gps` equal to `obs_dim`) with that of the
```

Comment:
nit: You should either use sphinx annotations for linkable objects
```
:class:`LinearlyCoupledMaternGP`
```
or double ticks to trigger code font
```
``num_gps`` equal to ``obs_dim``
```